main.cpp

```diff
@@ -222,6 +222,8 @@ struct Bakery
     int createTimer(int timeAmount);
 
     RyeBread breadToSell;
+
+    void sCNB();
```

Comment:
Name things what they are or what they do
do not use anacronyms in function names.
- [ ] revise